tests/integration/steps_multi_consumer_delegation.go

```diff
@@ -0,0 +1,305 @@
+package main
+
+// stepsDelegate tests basic delegation and resulting validator power changes.
+func stepsMultiConsumerDelegate(consumer1, consumer2 string) []Step {
```

Comment:
Could we make this a variadic function?

Reply:
Let's leave it at just 2 consumers for now.

Each chain has it's own channel and consumer states can differ depending on when the relaying happens in the tests. I would not go there at this point.